rllib/algorithms/algorithm_config.py

```diff
@@ -2396,7 +2400,7 @@ def offline_data(
         shuffle_buffer_size: Optional[int] = NotProvided,
         output: Optional[str] = NotProvided,
         output_config: Optional[Dict] = NotProvided,
-        output_compress_columns: Optional[bool] = NotProvided,
+        output_compress_columns: Optional[List[str]] = NotProvided,
```

Comment:
Now that this is a list[str], do we have to change the `Args:` description for this arg below?

Reply:
Goo dcatch again. Will check this, too!

Reply:
Ah, I see, I think this was already correct.